ui/component/app/view.jsx

```diff
@@ -249,6 +250,7 @@ function App(props: Props) {
 
     if (hasMyChannels) {
       fetchModBlockedList();
+      fetchCreatorSettings();
```

Comment:
Does this need to be fetched on app startup? Could it just be fetched on the creator settings page?